docs/GettingStartedGuide.md

```diff
@@ -33,7 +33,7 @@ The following two authentication approaches are available within your applicatio
 
 2. Using your **app access key id** and **secret access key**, which you can find on platform's [Apps & Keys](https://platform.here.com/profile/apps-and-keys) page. For more details on how to create your authentication and authorization client for retrieving HERE tokens, see [Authentication and Authorization Developer's Guide](https://developer.here.com/olp/documentation/access_control/topics/introduction.html).
 
-After you set up the credentials, you can then use them to access HERE OLP SDK libraries.
+After you set up the credentials, you can use them to access HERE OLP SDK libraries.
 
 ## <a name="components"></a> Available Components
 
```

Comment:
Comment to lines 40-45
Please change to:
"The OLP SDK for TypeScript package contains four independent modules that focus on a specific set of responsibilities:

- `olp-sdk-authentification`: gets OAuth2 bearer tokens used to confirm OLP requests.
- `olp-sdk-dataservice-api`: retrieves data from OLP.
- `olp-sdk-dataservice-read`: retrieves layer and partition data from OLP catalogs.
- `olp-sdk-fetch`: adds a subset of the [fetch](https://fetch.spec.whatwg.org/) API for [Node.js](https://nodejs.org/).